CMakeLists.txt

```diff
@@ -247,7 +246,7 @@ find_package(fmt "8.1.1" REQUIRED
   HINTS "${IRODS_EXTERNALS_FULLPATH_FMT}")
 find_package(spdlog "1.9.2" REQUIRED
   HINTS "${IRODS_EXTERNALS_FULLPATH_SPDLOG}")
-
+find_package(LibArchive "3.3.1" REQUIRED)
```

Comment:
Our externals package is at 3.5.2, but this is 3.3.1?

Why 3.3.1 instead of 3.5.2?

Reply:
Because we don't actually need 3.5.2, and some of the distros we support have an older version.

Reply:
Got it.